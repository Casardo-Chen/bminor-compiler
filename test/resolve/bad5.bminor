// redefinition

main: function void () = {
    x: integer = 5;
    x: integer =10;
    return 0;

}