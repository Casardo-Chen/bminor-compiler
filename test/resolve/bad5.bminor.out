resolve error: duplicate definitions of x in the same scope is invalid.
1 resolve error(s) found.
